OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -151,9 +192,10 @@ open class OptimoveConfigBuilder: NSObject {
                 inAppConsentStrategy: _inAppConsentStrategy,
                 inAppDeepLinkHandlerBlock: _inAppDeepLinkHandlerBlock,
                 pushOpenedHandlerBlock: _pushOpenedHandlerBlock,
-                _pushReceivedInForegroundHandlerBlock: _pushReceivedInForegroundHandlerBlock,
+                pushReceivedInForegroundHandlerBlock: _pushReceivedInForegroundHandlerBlock as Any,
```

Comment:
Why did we add the cast here, is it required?

Reply:
_pushReceivedInForegroundHandlerBlock was not annotated as optional
Fixed